lhotse/kaldi.py

```diff
@@ -248,7 +254,11 @@ def export_to_kaldi(
     )
     # reco2dur
     save_kaldi_text_mapping(
-        data={recording.id: recording.duration for recording in recordings},
+        data={
+            f"{recording.id}_{channel}": recording.duration
+            for recording in recordings
+            for channel in recording.sources[0].channels
```

Comment:
Should probably iterate over sources here too?

Reply:
I think we have an assert in the beginning to allow only single-source recordings, no?

Reply:
Oh I see. I think your changes are very close to supporting that but it’s fine to leave as-is.